Cargo.toml

```diff
@@ -28,6 +28,7 @@ serde_json = ">=1.0"
 sha2 = ">=0.10"
 tokio = { version = "1.17.0", features = ["rt-multi-thread"], optional = true }
 tonic = { version = ">=0.8.0", optional = true }
+attestation_agent = { git = "https://github.com/xhuan28/attestation-agent", branch = "dev"}
```

Comment:
According to review comments for the recent PR in enclave-cc, it is recommended to link the original github project. So please link "https://github.com/confidential-containers/attestation-agent" instead and I will upstream my changes there.